app.js

```diff
@@ -36,6 +46,18 @@ app.get('/', function(req, res, next) {
         res.status(200).json(data);
 	});
 });
+
+app.get('/login', function(req, res,next) {
+  console.log(req.body);
+  res.render('loginform', { title: 'Login to Petcare' });
+});
+
+
+
+app.post('/login', passport.authenticate('local', {successRedirect :'/signup',
```

Comment:
need the POST for logout too. Also need to make sure that the cookie/session is destroyed when user hits logout.